src/Microsoft.ML.Data/Data/IColumn.cs

```diff
@@ -97,6 +97,30 @@ private static IColumn GetColumnCore<T>(IRow row, int col)
             return new RowWrap<T>(row, col);
         }
 
+        /// <summary>
+        /// Exposes a single column in a schema. It column is considered inactive.
```

Comment:
> It column  [](start = 48, length = 11)

drop 'It'?

Reply:
I think I mean the...

---
In reply to: [215004572](https://github.com/dotnet/machinelearning/pull/778#discussion_r215004572) [](ancestors = 215004572)